packages/react-bootstrap-table2/src/text-editor.js

```diff
@@ -3,16 +3,23 @@ import React, { Component } from 'react';
 
 class TextEditor extends Component {
   componentDidMount() {
+    const { defaultValue } = this.props;
+    this.text.value = defaultValue;
     this.text.focus();
   }
 
   render() {
+    const {
+      onKeyDown,
+      onBlur
+    } = this.props;
```

Comment:
@AllenFang , good decision.  
I have to switch to develop branch and make changes there and push it?

i don't undertand what I did wrong ????

Reply:
Actually, I already help you to switch branch to `develop`.  but seems like you merge the upstream instead of rebase. so in this PR, you have my commits...  but you open another PR is good for me. let's decline this PR, thanks